diff --git i/kubetail w/kubetail
index f3deb53..45397f0 100755
--- i/kubetail
+++ w/kubetail
@@ -38,3 +38,3 @@ default_colored_output="${KUBETAIL_COLORED_OUTPUT:-$terminal_aware_default_color
 default_timestamps="${KUBETAIL_TIMESTAMPS:-}"
-default_jq_selector="${KUBETAIL_JQ_SELECTOR:-}"
+default_fx_selector="${KUBETAIL_JQ_SELECTOR:-}"
 default_skip_colors="${KUBETAIL_SKIP_COLORS:-7,8}"
@@ -49,3 +49,3 @@ colored_output="${default_colored_output}"
 timestamps="${default_timestamps}"
-jq_selector="${default_jq_selector}"
+fx_selector="${default_fx_selector}"
 skip_colors="${default_skip_colors}"
@@ -86,4 +86,4 @@ where:
     -e, --regex             The type of name matching to use (regex|substring). Defaults to ${regex}.
-    -j, --jq                If your output is json - use this jq-selector to parse it. Defaults to \"${default_jq_selector}\".
-                            example: --jq \".logger + \\\" \\\" + .message\"
+    -j, --fx                If your output is json - use this fx-selector to parse it. Defaults to \"${default_fx_selector}\".
+                            example: --fx \".logger + \\\" \\\" + .message\"
     -k, --colored-output    Use colored output (pod|line|false).
@@ -198,7 +198,7 @@ if [ "$#" -ne 0 ]; then
 			;;
-		-j|--jq)
+		-j|--fx)
 			if [ -z "$2" ]; then
-				jq_selector="${default_jq_selector}"
+				fx_selector="${default_fx_selector}"
 			else
-				jq_selector="$2"
+				fx_selector="$2"
 			fi
@@ -384,6 +384,6 @@ for pod in ${matching_pods[@]}; do
 		colorify_lines_cmd="while read -r; do echo \"$colored_line\" | tail -n +1; done"
-		if [ "z" == "z$jq_selector" ]; then
+		if [ "z" == "z$fx_selector" ]; then
 			logs_commands+=("${kubectl_cmd} ${timestamps} | ${colorify_lines_cmd}");
 		else
-			logs_commands+=("${kubectl_cmd} | jq --unbuffered -r -R --stream '. as \$line | try (fromjson | $jq_selector) catch \$line' | ${colorify_lines_cmd}");
+			logs_commands+=("${kubectl_cmd} | fx '$fx_selector' | ${colorify_lines_cmd}");
 		fi
